frontend/components/sections/lead-form.js

```diff
@@ -0,0 +1,96 @@
+import Button from '../elements/button'
+import { useState } from 'react'
+import { fetchAPI } from 'utils/api'
+import * as yup from 'yup'
+import { Formik } from 'formik'
+
+const LeadForm = ({ data }) => {
+  const [loading, setLoading] = useState(false)
+
+  const schema = yup.object().shape({
+    email: yup.string().email().required(),
+  })
+
+  return (
+    <div className="py-10 text-center">
+      <h1 className="text-3xl mb-10 font-bold mb-2">{data.title}</h1>
+      <div className="flex flex-col items-center flex-wrap ">
+        <Formik
+          initialValues={{ email: '' }}
+          validate={async (values) => {
```

Comment:
With formik and yup you don't need to develop the validate function. You can directly use the [`validationSchema`](https://formik.org/docs/guides/validation#validationschema) prop.